SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+## Supported Versions
+
+| Version   | Supported          |
+| -------   | ------------------ |
+|  0.3.0    | :white_check_mark: |
+| < 0.3.0   | :x:                |
+
+## Reporting a Vulnerability
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them via [email](info@auto-dl.tech).
```

Comment:
when I click on this link it gives a 404.
Why don't we just put the email id here ?

Reply:
Oww seriously?

Reply:
> Why don't we just put the email id here ?
Look cool :P